src/ImageSharp/Image/PixelAccessor{TColor}.cs

```diff
@@ -186,8 +186,7 @@ public void CopyFrom(PixelArea<TColor> area, int targetY, int targetX = 0)
             int width = Math.Min(area.Width, this.Width - targetX);
             int height = Math.Min(area.Height, this.Height - targetY);
 
-            this.CheckDimensions(width, height);
-
+            // this.CheckDimensions(width, height); TODO: Why was width == 0 or height == 0 considered a problem? Copy implementations do not fail on this (just do nothing)!
```

Comment:
The intention is that the developer should get a notification when they call the method with an invalid x and y offset. Calling this method with an x or y offset that does nothing is also considered invalid.  The caller should add a check for this instead of doing a non-copy inside this class.